pipeline/extract/document.py

```diff
@@ -123,7 +127,30 @@ def save_text(self, text_filepath: Path):
         with open(text_filepath, "wt") as f:
             f.write(self.to_string())
 
+    @classmethod
+    def from_dict(cls, json_dict: dict) -> "Document":
+        """Load a document from a dictionary"""
+        for page in json_dict["pages"]:
+            page["text_blocks"] = [TextBlock(**block) for block in page["text_blocks"]]
+        return cls(
+            pages=[Page(**page) for page in json_dict["pages"]],
+            filename=json_dict["filename"],
+        )
+
     @classmethod
```

Comment:
Implement method to convert from json to document.

Reply:
Could this use `from_dict` after `data = json.load(f)`?